docs/developing_iree/e2e_benchmarking.md

```diff
@@ -348,6 +348,26 @@ adb shell taskset f0 /data/local/tmp/benchmark_model_plus_flex \
   --enable_op_profiling=true
 ```
 
+```shell
+# Benchmark with TFLite running on GPU
+adb shell taskset f0 /data/local/tmp/benchmark_model \
+  --graph=/data/local/tmp/MatrixOpsStaticModule/tflite/matmul_lhs_batch.tflite \
+  --warmup_runs=1 \
+  --num_threads=1 \
+  --num_runs=10 \
+  --enable_op_profiling=true \
+  --use_gpu=true
+```
+
+Running benchmark on GPU won't give op profiling. To detailed profiling
+information for GPU you can run the following script:
+
+```shell
+# Op profiling on GPU using OpenCL backend
+sh //tensorflow/lite//delegates/gpu/cl/testing/run_performance_profiling.sh \
```

Comment:
I don't understand this path – is it on the phone somehow?

Reply:
It is a script in tensorflow repo. The script calls adb to transfer the files and run the benchmark. Did I get the path to tensorflow repo wrong?

Reply:
Assuming this is being run from within `iree/third_party/tensorflow`, this would be the path:
```suggestion
sh tensorflow/lite/delegates/gpu/cl/testing/run_performance_profiling.sh \
```

Reply:
I see, thanks for fixing this. Should the `iree/third_party/tensorflow` be added? I'll leave it as you suggested for now.